sdks/go/pkg/beam/core/graph/window/trigger.go

```diff
@@ -35,3 +35,38 @@ const (
 	NeverTrigger                           string = "Trigger_Never_"
 	AfterSynchronizedProcessingTimeTrigger string = "Trigger_AfterSynchronizedProcessingTime_"
 )
+
+// TriggerDefault constructs a default trigger that fires after the end of window.
+// No provision for late arriving data.
```

Comment:
Does this mean it discards late data, or is it unspecified? Does that mean it fires once?

Reply:
Yes it fires once. I haven't tested this for late data as I wasn't able to advance processing time on flink runner.